src/main/java/uk/gov/hmcts/darts/annotation/service/impl/AnnotationDownloadServiceImpl.java

```diff
@@ -44,14 +43,15 @@ public AnnotationResponseDto downloadAnnotationDoc(Integer annotationId, Integer
             throw new DartsApiException(INVALID_ANNOTATIONID_OR_ANNOTATION_DOCUMENTID);
         }
 
-        final ExternalObjectDirectoryEntity externalObjectDirectoryEntity = eodDir.get(0);
+        final ExternalObjectDirectoryEntity latestExternalObjectDirectoryEntity = eodDir.get(0);
 
-        final InputStreamResource blobStream = annotationDataManagement.download(externalObjectDirectoryEntity);
+        final InputStreamResource blobStream = annotationDataManagement.download(latestExternalObjectDirectoryEntity);
```

Comment:
If you compare with Transcription download - you'll notice that we actually send all the external_object_directory 's to the Facade to do the "heavy lifting". This was the design that @hemantasharma1129 wanted as I first had something similar when I started a similar integration in [DMP-656](https://tools.hmcts.net/jira/browse/DMP-656)